README.md

```diff
@@ -109,6 +109,16 @@ You can then deploy to IPFS like so:
 yarn deploy:ipfs:smoldot-browser-demo
 ```
 
+## Run the Multi Network browser demo
+The Multiple Network functionality enables connecting to multiple networks from a single web-page.
+Running the following command will build all necessary dependencies and run the Multiple Network browser demo. It will be served on https://localhost:1234/
+
+```bash
+yarn dev:multiple-network-demo 
+```
+
+(Make sure to run `$ yarn install` before.)
+
```

Comment:
It would be better to move this to line 70 (just after `## Run local version of the Smoldot browser demo`) so that all demos will be gathered together

Reply:
Done ✅